assets/app/views/browse/pod.html

```diff
@@ -95,7 +95,7 @@
                         context="projectContext"
                         options="logOptions"
                         status="pod.status.phase"
```

Comment:
I'll comment that out for now.  I'm not going to do a `Fixes #7414` in this PR description because I think we need a bit more on that issue, right?